jwst/extract_1d/extract.py

```diff
@@ -2530,7 +2531,7 @@ def run_extract1d(input_model, refname, smoothing_length, bkg_order,
     input_model : data model
         The input science model.
 
-    refname : str
+    extract_ref_name : str
         The name of the reference file, or "N/A".
```

Comment:
Perhaps expand this to "The name of the EXTRACT1D reference file", because there's no longer a single ("the") reference file for the step.